src/main/java/by/onliner/recorder/Recorder.java

```diff
@@ -0,0 +1,50 @@
+package by.onliner.recorder;
+
+import org.monte.media.Format;
+import org.monte.media.Registry;
+import org.monte.screenrecorder.ScreenRecorder;
+
+import java.awt.*;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Recorder extends ScreenRecorder {
```

Comment:
Do you need this class as you have VideoRecorder?

Reply:
Yes, i need this class like initializatior of my VideoRecorder